.gitlab-ci-check-commits.yml

```diff
@@ -3,8 +3,10 @@ stages:
   - test
 
 test:check-commits:
-  image: alpine
   stage: test
+  except:
+    - master
```

Comment:
Should these actually have the famous `/^(master|[0-9]+\.[0-9]+\.x)$/` expression? Not sure if we run nightly on those, but definitely they have the same limitation as the master branch: Commit checks are useless on them.

Reply:
Yes! It is not critical but rather a good practice. I'll fix.